### ISSUE TITLE
0.5.0 broke nssm

### ISSUE THREAD

**Issue by ohitstom** [hrfee/jfa-go#302]
![image](https://github.com/hrfee/jfa-go/assets/22730962/dc634e7e-2f1b-4dac-8de4-dd6746033137)
![image](https://github.com/hrfee/jfa-go/assets/22730962/1350d1ae-61fd-4697-8ebc-74008b82ca65)

```ERROR systray: systray_windows.go:769 Unable to init instance: Unspecified error [error=Unable to init instance: %v error_location=github.com/getlantern/systray.registerSystray (systray_windows.go:769) error_text=Unable to init instance: Unspecified error error_type=errors.Error]
ERROR systray: systray_windows.go:769   at github.com/getlantern/systray.registerSystray (systray_windows.go:769)
ERROR systray: systray_windows.go:769   at github.com/getlantern/systray.Register (systray.go:106)
ERROR systray: systray_windows.go:769   at github.com/getlantern/systray.Run (systray.go:77)
ERROR systray: systray_windows.go:769   at main.RunTray (tray.go:19)
ERROR systray: systray_windows.go:769   at main.main (main.go:730)
ERROR systray: systray_windows.go:769   at runtime.main (proc.go:250)
ERROR systray: systray_windows.go:769   at runtime.goexit (asm_amd64.s:1598)
ERROR systray: systray_windows.go:769 Caused by: Unspecified error```

**Comment by ohitstom** [comment]
![image](https://github.com/hrfee/jfa-go/assets/22730962/26b9164c-2ca9-4121-b33c-d100eacfc06a)
i get no login prompt over proxy

**Comment by hrfee** [comment]
which release? 5.0 or an unstable one?

**Comment by hrfee** [comment]
If it's an unstable, and you're happy with keeping whatever data you had before 5.0, just delete/move the DB and it will regenerate from the old JSON files.

**Comment by ohitstom** [comment]
0.5.0, most of these issues ive managed to fix with tinkering and brute forcing the settings by setting them over and over and over.

however my ombi will not connect no matter what, localhost or otherwise, invites also no longer display my server invite

**Comment by ohitstom** [comment]
> If it's an unstable, and you're happy with keeping whatever data you had before 5.0, just delete/move the DB and it will regenerate from the old JSON files.

already tried this it just breaks whenever i do this lol

**Comment by hrfee** [comment]
Alright, well for the time being if you don't want issues you'll have to downgrade, again data should be kept from the JSON files. I'm not gonna be available to do any work or troubleshooting until next Thursday, apologies, but if you figure out anything that might be helpful or get any logs in that time, put them here.

**Comment by ohitstom** [comment]
> Reference in new iss

no worries its just super stressful thats all i expected a seamless update lol

for some reason in this new update nssm doesnt work either? is there an option for a trayless executable on windows
and again i cant downgrade it reset/broke my previous db and for some reason my config got reset too...

**Comment by hrfee** [comment]
For NSSM: Was gonna say to try a non-tray version as you're daemonizing it, but forgot I don't build those automatically for Windows, I'll add one shortly.

The config thing: Did your actual config file get overwritten/deleted? That seems very unlikely, only time a template config file is made is when a config file doesn't exist. I saw in your screenshot that your `db` folder and `config.ini` are in the same directory as the executable, by default they should be in `%AppData%/jfa-go`, so check there to see if you can find the original. If you've put them in this other directory yourself, you need to make sure jfa-go knows that by adding `-d "C:\Program Files\Jellyfin\Jellyfin Accounts"` to the "Arguments" section in NSSM.

DB "Nuked": I'm not 100% on what you mean by this, but I just fixed issue #303 where user details (emails, discord, etc.) were being deleted if jfa-go couldn't connect to Jellyfin. Whether that was the issue for you, I don't know, but try out the latest unstable and let me know what's still broken.

**Comment by hrfee** [comment]
Oh, and for the proxy: If you're reverse proxying to a subfolder (i.e. accessing jfa-go via my-jellyfin.com/accounts), make sure "Reverse Proxy subfolder" in Settings > General is set (for the example given, you'd set if to `/accounts`). If that doesn't fix the issue/you're using a subdomain, send a copy of your proxy config so I can try to replicate.

**Comment by ohitstom** [comment]
> Oh, and for the proxy: If you're reverse proxying to a subfolder (i.e. accessing jfa-go via my-jellyfin.com/accounts), make sure "Reverse Proxy subfolder" in Settings > General is set (for the example given, you'd set if to `/accounts`). If that doesn't fix the issue/you're using a subdomain, send a copy of your proxy config so I can try to replicate.

my proxy is to a simple subdomain, as for the settings and db wiping itself when I tried to update through the web UI I got a nasty error about permissions then jfa itself crashed with a long error string

the db and settings weren't deleted per say since the files were there but something must of got corrupted.

as for the db being in the same folder as the executable that was just temporary, usually they are in the default location!

if you could compile a tray less version that should fix nssm and would be great thank you

**Comment by hrfee** [comment]
the non tray versions are available now. I think the updating through the web UI might be an issue with where your executable is, maybe it doesn't have permissions to modify itself there? The proxy thing i'm not sure about, try force-refreshing the page, and if that doesn't help, open the Network tab in the browser's developer options and reload the page, then look out for any errors.

**Comment by ohitstom** [comment]
fixed via #53c9569a371bd6bc8b13d71d0eae2610525558d6
